finals_code/main.swift

```diff
@@ -22,13 +22,13 @@
 // -~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~
 
 // create a class named Account
-class account 
+class Account 
 {
  var username:String         // declare username variable [String]
```

Comment:
The comments are redundant. Refrain from commenting the same phrase on the same methods.